_episodes/01-section1-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Section 1: Environment For Collaborative Code Development"
+colour: "#f7d77e"
```

Comment:
I'd go for a more pastel, lighter shade of mustard to go with the current colour scheme - maybe #fcecc0 ?

Reply:
Sounds good 👍 .